pulsar-client-admin-api/src/main/java/org/apache/pulsar/client/admin/GetTopicsMode.java

```diff
@@ -0,0 +1,61 @@
+/**
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+package org.apache.pulsar.client.admin;
+
+/**
+ * Pulsar supports persistent topic and non-persistent topic.
+ * The GetTopicsMode is used to get persistent/non-persistent topics.
+ */
+public enum GetTopicsMode {
```

Comment:
Maybe we can reuse the `TopicType` enumeration, when the type is not set, then read all. This is just a suggestion.

Reply:
TopicType is partitioned/non-partitioned. Maybe change to TopicDomain is better.